### PR TITLE
Adjust spacing for contributor.

### LINKED ISSUE #1988: Refine spacing of ORCID UI
Current
image

Proposed design
image

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1988

## Why was this change made?
Per ticket

## How was this change tested?
image

### CODE REVIEW

File: app/components/works/contributor_row_component.html.erb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+<div class="row inner-container contributor-row" data-controller="contributors" data-contributors-required-value="<%= author? %>">
   <div class="col-md-11">
     <div class="row">
       <div class="col-md-3">
         <%= form.label :role_term, role_term_label, class: 'form-label' %>
         <%= render PopoverComponent.new key: 'work.role_term' %>
         <%= select_role %>
       </div>
-      <div class="col-md-1 role-prefix">
+      <div class="col-sm-1 role-prefix">
```
Comment (jcoyne) [inline app/components/works/contributor_row_component.html.erb:9]: This switch causes this to get awkwardly positioned:

Screen Shot 2021-08-23 at 12 17 07 PM

It looked like this before.
Screen Shot 2021-08-23 at 12 15 06 PM

File: app/packs/stylesheets/workEditor.scss [hunk 2]
```diff
@@ -74,4 +69,24 @@
   div.embargo-date {
     margin-bottom: 1rem !important;
   }
+
+  .contributor-row.inner-container {
```
Comment (jcoyne) [inline app/packs/stylesheets/workEditor.scss:73]: Is it necessary to have `.inner-container` here? Why can't we just put this under `.contributor-row`?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/components/works/contributor_row_component.html.erb (hunk 1 above)
Comment (justinlittman) [reply app/components/works/contributor_row_component.html.erb:9]: Switched back to `col-md-1`.

File: app/packs/stylesheets/workEditor.scss (hunk 2 above)
Comment (justinlittman) [reply app/packs/stylesheets/workEditor.scss:73]: It gets trumped by `.work-editor section .inner-container` in `editor.scss`, which sets a different margin.

Comment (jcoyne) [reply app/packs/stylesheets/workEditor.scss:73]: Please add a comment about this so there's no accidental refactoring to something simpler.

Comment (justinlittman) [reply app/packs/stylesheets/workEditor.scss:74]: Done.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**